cmd/codegen/generator/open_api.go

```diff
@@ -426,9 +427,13 @@ func getOutputPackageForOpenApi(pkg string, groupWithVersion []string, outputPac
 	if version == "" {
 		version = "unversioned"
 	}
-	return filepath.Join(outputPackage, "openapi", fmt.Sprintf("%s_%s_%s", packageToDirName(pkg),
-		groupWithVersion[0],
-		version)), nil
+	return filepath.Join(outputPackage, "openapi", fmt.Sprintf("%s_%s_%s", toValidPackageName(packageToDirName(pkg)),
```

Comment:
Is this a new issue or still the release build failing? Just wondering whether this is something new or whether it has not worked at all yet?